src/pages/home/home.scss

```diff
@@ -0,0 +1,74 @@
+@import "~theme/base/variables";
+
+.homeContainer {
+  height: 100%;
+}
+
+.navigationBar {
+  padding: 1.25em 0em;
+}
+
+.navigationBar .leftNavLinks {
```

Comment:
I think `.navigationBar` is not needed, just defining `.leftNavLinks` should be enough, the reason for this is because when using css modules, `.leftNavLinks` will be compiled to `.xyz123` when going to production, webpack guarantees the class has a unique name.

Reply:
This has been resolved in the latest push.